client/src/components/addCenterComponent.js

```diff
@@ -0,0 +1,234 @@
+import React from 'react';
+import { ModalHeader } from './modalHeader';
+import { FormGroup } from './formGroup';
+import centerNameIcon from '../resources/images/glyphicons-21-home.png';
+import centerImageIcon from '../resources/images/glyphicons-139-picture.png';
+import addressIcon from '../resources/images/glyphicons-243-map-marker.png';
+import cityIcon from '../resources/images/glyphicons-740-cadastral-map.png';
+import capacityIcon from '../resources/images/glyphicons-44-group.png';
+import amountIcon from '../resources/images/glyphicons-228-usd.png';
```

Comment:
You could create an `<Icon>` component that simply takes a prop of the image/icon name and returns the Icon. This allows you to import only `Icon` and use where necessary rather than import 6 or more individual icons.